primitives/core/Cargo.toml

```diff
@@ -68,7 +68,7 @@ twox-hash = { version = "1.6.1", default-features = false, optional = true }
 libsecp256k1 = { version = "0.6", default-features = false, features = ["hmac", "static-context"], optional = true }
 sp-core-hashing = { version = "4.0.0-dev", path = "./hashing", default-features = false, optional = true }
 merlin = { version = "2.0", default-features = false, optional = true }
-ss58-registry = "1.0.0"
+ss58-registry = "1.5.0"
```

Comment:
```suggestion
ss58-registry = { version = "1.5.0", default-features = false }
```

Reply:
And then you also need to enable the `std` feature below.

Reply:
ah good point.